releasenotes/notes/add-elide-swaps-b0a4c373c9af1efd.yaml

```diff
@@ -0,0 +1,35 @@
+---
+features:
+  - |
+    Added a new optimization transpiler pass, :class:`~.ElideSwaps`,
+    which is designed to run prior to the :ref:`layout_stage` and will
+    optimize away any :class:`~.SwapGate`\s in a circuit by permuting virtual
+    qubits. For example, taking a circuit with :class:`~.SwapGate`\s:
+
+    .. plot::
+
```

Comment:
If I remember right, doesn't `.. plot::` need an `:include-source:` option?  We might consider defaulting that to `true` in the docs config, and manually overriding it to `false` where appropriate.

Reply:
It does if you want to include the source in the display. My intent here was to just show the circuit diagram for the first plot and include the source for the second (although I forgot that there)

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/9523/commits/8f699e86e1c3aba1dc3102f6c6aeb4433ca05777